package.json

```diff
@@ -103,7 +103,7 @@
     "core-js": "3.7.0",
     "flexboxgrid": "6.3.1",
     "focus-trap": "6.1.4",
-    "fundamental-styles": "0.14.0",
+    "fundamental-styles": "0.14.1-rc.28",
```

Comment:
Could you bring compiled `quick-view.css` code from fundamental-styles dist into `quick-view.component.scss`, instead of increasing styles version?